app/src/main/java/org/fossasia/openevent/general/event/topic/SimilarEventsFragment.kt

```diff
@@ -153,4 +168,31 @@ class SimilarEventsFragment : Fragment() {
         moreLikeThis.isGone = similarEvents.isEmpty()
         similarEventsRecycler.isGone = similarEvents.isEmpty()
     }
+
+    private fun setUpAdapter() {
+        similarEvents.clear()
+        var id: Long
+        var add: Boolean
+        if (similarIdEvents.size != 0 && similarLocationEvents.size != 0) {
```

Comment:
Use `when` expression when there  is more than `if else` expression

Reply:
I don't think `when` can be used on 2 variables. Please correct me if I'm wrong

Reply:
yes it could . for example
```
when{
   variable1!=0 && variable!=0 ->{ do somwthing here}
    variable3==2 ->{ do something here}
}
```

Reply:
resolved